### PR TITLE
Support supplied cert/key pair supplied as env vars

### PR DESCRIPTION
**Pull request by joel-wright**
# Description

Adds support for serialising vespa key/cert files to disk at startup supplied as base64 encoded env vars.

## Type of change

Please select the option(s) below that are most relevant:

- [ ] Bug fix
- [x] New feature
- [ ] Breaking change

## How Has This Been Tested?

Ad-hoc startup tests

## Reviewer Checklist

- [ ] The PR represents a single feature (small driveby fixes are also ok)
- [ ] The PR includes tests that are sufficient for the level of risk
- [ ] The code is sufficiently commented, particularly in hard-to-understand areas
- [ ] Any required documentation updates have been made
- [ ] Any TODOs added are captured in future tickets
- [ ] No FIXMEs remain

### CODE REVIEW

File: startup.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
```
Comment (diversemix) [inline startup.sh:2]: do we want to have the standard `set ueo pipefail` ?

**Review by diversemix — APPROVED** [review]
lgtm

File: startup.sh (hunk 1 above)
Comment (joel-wright) [reply startup.sh:2]: I'll need to do a bit more work setting up the vars, but sure

Comment (joel-wright) [reply startup.sh:2]: done

**Review by diversemix — APPROVED** [review]

**Pull request merged**